engine/apps/grafana_plugin/apps.py

```diff
@@ -25,13 +26,16 @@ def ready(self):
         # TODO: this logic should probably be moved out to a common utility
         is_not_migration_script = any(startup_command in sys.argv for startup_command in STARTUP_COMMANDS)
         if is_not_migration_script and settings.IS_OPEN_SOURCE:
-            Organization = apps.get_model("user_management", "Organization")
-            has_existing_org = Organization.objects.first() is not None
-
-            # only enforce the following for new setups - if no organization exists in the database
-            # and the GRAFANA_API_URL env var is not specified, exit the application
-            if has_existing_org is False and settings.SELF_HOSTED_SETTINGS["GRAFANA_API_URL"] is None:
-                logger.error(
-                    f"For OSS installations, GRAFANA_API_URL is a required environment variable. Please set it and restart the application."
-                )
-                sys.exit()
+            try:
+                Organization = apps.get_model("user_management", "Organization")
+                has_existing_org = Organization.objects.first() is not None
```

Comment:
this line makes a db call.. just ignore this validation if the db is unavailable..